plugins/bticino/src/main.ts

```diff
@@ -0,0 +1,381 @@
+                    if (!isStunMessage(message)) {
+                        const isRtpMessage = isRtpMessagePayloadType(getPayloadType(message));
+                        if (!isRtpMessage)
+                            return;
+                        vseen++;
+                        rtsp.sendTrack(videoTrack, message, !isRtpMessage);
+                        const seq = getSequenceNumber(message);
+                        if (seq !== (vseq + 1) % 0x0FFFF)
+                            vlost++;
+                        vseq = seq;
+                    }
+                });
+
+                sip.videoRtcpSplitter.on('message', message => {
+                    rtsp.sendTrack(videoTrack, message, true);
+                });
+                
+                sip.audioSplitter.on('message', message => {
+                    if (!isStunMessage(message)) {
+                        const isRtpMessage = isRtpMessagePayloadType(getPayloadType(message));
+                        if (!isRtpMessage)
+                            return;
+                        aseen++;
+                        rtsp.sendTrack(audioTrack, message, !isRtpMessage);
+                        const seq = getSequenceNumber(message);
+                        if (seq !== (aseq + 1) % 0x0FFFF)
+                            alost++;
+                        aseq = seq;
+                    }
+                });
+
+                sip.audioRtcpSplitter.on('message', message => {
+                    rtsp.sendTrack(audioTrack, message, true);
+                });
+
+                this.session = sip;
+
+                try {
+                    await rtsp.handleTeardown();
+                    this.log.d('rtsp client ended');
+                }
+                catch (e) {
+                    this.log.e('rtsp client ended ungracefully' + e);
+                }
+                finally {
+                    cleanup();
+                }
+            }
+            catch (e) {
+                sip?.stop();
+                throw e;
+            }
+        });
+
+        this.resetStreamTimeout();
+
+        const mediaStreamOptions = Object.assign(this.getSipMediaStreamOptions(), {
+            refreshAt: Date.now() + STREAM_TIMEOUT,
+        });
+
+        const mediaStreamUrl: MediaStreamUrl = {
+            url: playbackUrl,
+            mediaStreamOptions,
+        };
+        this.currentMedia = mediaStreamUrl;
+        this.currentMediaMimeType = ScryptedMimeTypes.MediaStreamUrl;
+
+        return mediaManager.createMediaObject(mediaStreamUrl, ScryptedMimeTypes.MediaStreamUrl);
+    }
+
+    getSipMediaStreamOptions(): ResponseMediaStreamOptions {
+        return {
+            id: 'sip',
+            name: 'SIP',
+            // this stream is NOT scrypted blessed due to wackiness in the h264 stream.
+            // tool: "scrypted",
+            container: 'sdp',
+            video: {
+                codec: 'h264',
+                h264Info: {
```

Comment:
is this accurate or did you copy this from somewhere? May be better to leave it blank and let rebroadcast handle it.

Reply:
It was a shameless copy from "sip". I tried just remove the h264Info part but that gave me following errors:

```
[BTicino SIP Plugin] unknown nal unit type 31

[BTicino SIP Plugin] unknown nal unit type 0

[BTicino SIP Plugin] unknown nal unit type 27

[BTicino SIP Plugin] unknown nal unit type 25

[BTicino SIP Plugin] unknown nal unit type 27
```
So I opted to remove it completely which works.